certify-service/src/main/resources/application-local.properties

```diff
@@ -159,7 +159,7 @@ mosip.certify.key-values={\
               'credential_configurations_supported' : { \
                  "InsuranceCredential" : {\
```

Comment:
Why we have mDL in InsuranceCredential config?

Reply:
InsuranceCredential and DrivingLicenseCredential are different elements in credential_configurations_supported obj

Reply:
The full file has both creds. LGTM